lib/cppcheck.cpp

```diff
@@ -688,16 +693,20 @@ unsigned int CppCheck::checkFile(const FileWithDetails& file, const std::string
             std::string locfile = Path::fromNativeSeparators(output.location.file());
             if (mSettings.relativePaths)
                 locfile = Path::getRelativePath(locfile, mSettings.basePaths);
-
-            ErrorMessage::FileLocation loc1(locfile, output.location.line, output.location.col);
-
-            ErrorMessage errmsg({std::move(loc1)},
-                                "", // TODO: is this correct?
-                                Severity::error,
-                                output.msg,
-                                "syntaxError",
-                                Certainty::normal);
-            reportErr(errmsg);
+            if (output.type == simplecpp::Output::Type::FILE_NOT_FOUND) {
+                const std::string fixedpath = Path::toNativeSeparators(file.path());
```

Comment:
isn't it better to use output.location.file() and warn about the file that simplecpp complains about